src/main/java/amazonreviewsapp/springboot/service/ReviewService.java

```diff
@@ -14,5 +20,47 @@ public void save(Review review) {
         reviewRepository.save(review);
     }
 
+    public List<Object> findMostCommentedProducts() {
+        return reviewRepository.findMostCommentedProducts();
+    };
 
+    public Object[] findMostUsedWordsFromReviews() {
+        List<String> texts =
+                reviewRepository.findAll().stream()
+                        .map(Review::getText)
+                        .collect(Collectors.toList());
+        HashMap<String, Integer> rating = new HashMap<>();
+        for (String text: texts) {
+            iterateOverReview(text, rating);
+        }
+        return sortHashMap(rating);
+    }
+
+    private void iterateOverReview(String string, HashMap<String, Integer> hashMap) {
+        String[] split =
+                string.toLowerCase()
```

Comment:
Старайся не розтягувати свій код
```suggestion
               String[] split = string.toLowerCase()
```